cmake/FindLMDB.cmake

```diff
@@ -0,0 +1,28 @@
+# Try to find the LMBD libraries and headers
```

Comment:
Why do you need this file if using `find_package(LMDB CONFIG)`?

Reply:
I assumed you wanted compatibility for non-Hunter builds. I wouldn't do it if I hadn't seen the other find modules there.

Reply:
No needed. We keep Find modules only for packages not in Hunter.